modules/host/negotiatedownload.go

```diff
@@ -0,0 +1,230 @@
+package host
+
+import (
+	"errors"
+	"net"
+	"time"
+
+	"github.com/NebulousLabs/Sia/crypto"
+	"github.com/NebulousLabs/Sia/encoding"
+	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/types"
+)
+
+var (
+	// errLargeDownloadBatch is returned if the renter requests a download
+	// batch that exceeds the maximum batch size that the host will
+	// accomondate.
+	errLargeDownloadBatch = errors.New("a batch of download requests has exceeded the maximum allowed download size")
+
+	// errRequestOutOfBounds is returned when a download request is made which
+	// asks for elements of a sector which do not exist.
+	errRequestOutOfBounds = errors.New("a download request has been made that exceeds the sector boundaries")
```

Comment:
passive voice is uncommon for error messages. Usually it's more like "download request exceeds maximum batch size" and "download request has invalid bounds"